samples/web/content/apprtc/apprtc.py

```diff
@@ -495,12 +515,109 @@ def write_response(self, result, params, messages):
       'result': result,
       'params': params
     }))
+    
+  def report_error(self, error):
+    self.write_response(error, {}, [])
 
   def write_room_parameters(self, room_id, client_id, messages, is_initiator):
     params = get_room_parameters(self.request, room_id, client_id, is_initiator)
     self.write_response('SUCCESS', params, messages)
+    
+  def handle_call(self, msg, room_id):
```

Comment:
I agree with this, I was thinking about collisions when working on it and had the same thought. We could also internally prefix the room name with their ID or something similar to give each user their own namespace.